src/curated-corpus/integration-test-mocks/getScheduledSurfacesForUser.ts

```diff
@@ -25,7 +24,6 @@ const allScheduledSurfaces: ScheduledSurface[] = [
       ProspectType.TitleUrlModeled,
       ProspectType.RssLogistic,
       ProspectType.RssLogisticRecent,
-      ProspectType.SlateScheduler,
       ProspectType.SlateSchedulerV2,
```

Comment:
Unrelated to the main purpose of this PR - these two prospect types are no longer valid, removed them from the mocks